src/structures/paging/page.rs

```diff
@@ -332,7 +332,7 @@ impl<S: PageSize> fmt::Debug for PageRange<S> {
 }
 
 /// A range of pages with inclusive upper bound.
-#[derive(Clone, Copy, PartialEq, Eq)]
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Same point as above: I don't think that deriving `PartialOrd`/`Ord` makes sense for range types.